pkg/segment/query/processor/statscommand.go

```diff
@@ -39,13 +39,23 @@ type ErrorData struct {
 	notSupportedStatsType map[string]struct{} // columnName -> struct{}. Tracks unsupported stats types
 }
 
+type statsType uint8
+
+const (
+	InvalidCmd statsType = iota
+	SegmentStatsCmd
+	GroupByCmd
+)
```

Comment:
We already have this in segstructs.go; probably just reuse that instead of making a new enum
```
const (
	InvalidCmd QueryType = iota
	SegmentStatsCmd
	GroupByCmd
	RRCCmd
)
```

Reply:
Yeah I created a new enum because `RRCCmd` is not a valid option. So I guess, I will just use this.